apis/apis.json

```diff
@@ -2996,6 +2996,24 @@
         "Translation"
       ]
     },
+    {
+      "id": "Google.Cloud.VMMigration.V1",
```

Comment:
Should this one go after the Google.Cloud.Video* ones?

Reply:
Nope - the capital "M" means that under ordinal sorting, it goes before it. (This insertion was autogenerated, so should be okay.)